database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --for=create --timeout=60s
+kubectl -n ${NAMESPACE} wait pod --all --for=condition=ready --timeout=30s
+
+# Create the kubearchive database
+LOCAL_PORT=5433
+echo Forwarding port ${LOCAL_PORT} to service/kubearchive-wr:5432.
+export PGPASSWORD=$(kubectl -n ${NAMESPACE} get secret kubearchive-superuser -o jsonpath='{.data.password}' | base64 --decode)
+kubectl -n ${NAMESPACE} port-forward service/kubearchive-rw ${LOCAL_PORT}:5432 >& /dev/null &
+pid=$!
+
+echo Waiting for port $LOCAL_PORT to become available.
+while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
+    echo -n .
+    sleep 0.5
+done
+echo .
+psql -h localhost -U postgres -p ${LOCAL_PORT} -f kubearchive.sql
```

Comment:
This will require `psql` to be installed on the CI. I don't recommend to go that way, this is why I did the thing with the pod and `cat` into the database pod.

Reply:
Well, `psql` is already there and available for the CI. It's running now.

Either way, `psql` is part of the general PostgreSQL installation.  It's the main way to interact with the database.  It seems silly to go through extra hoops to do this when there's a command line tool that will do it for you.

And database admin are going to use `psql`, so it's more like the user experience.

Reply:
I saw it running but I didn't process that it was already installed, I tend to think about CI as some minimal environment but no, we are running with an image full of tools:

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md